app/util/parameter-parsing.ts

```diff
@@ -1,12 +1,33 @@
 import { RequestValidationError } from './errors';
 import HarmonyRequest from '../models/harmony-request';
 
+/**
+ * Tag class for denoting errors during parsing
+ *
+ */
+export class ParameterParseError extends Error {}
+
+/**
+  * Helper function for parameters that parses and validates boolean values. A null value
+  * defaults to false.
+  *
+  * @param valueStr - the unparsed boolean as it appears in the input
+  * @returns the parsed result
+  * @throws ParameterParserError - if there are errors while parsing
+  */
+export function parseBoolean(valueStr: string): boolean {
```

Comment:
should probably add a unit test for this

Reply:
Done. I feel like this is payback for me always asking you to add tests for my code.